channeldb/kvdb/etcd/db.go

```diff
@@ -50,7 +59,73 @@ func newEtcdBackend(config ConnConfig) (*db, error) {
 	return backend, nil
 }
 
-// Close closes the db, but closing the underlying etcd client connection.
+// View opens a database read transaction and executes the function f with the
+// transaction passed as a parameter.  After f exits, the transaction is rolled
+// back.  If f errors, its error is returned, not a rollback error (if any
+// occur).
+func (db *db) View(f func(tx walletdb.ReadTx) error) error {
+	apply := func(stm STM) error {
+		return f(newReadWriteTx(stm))
+	}
+
+	return RunSTM(db.cli, apply)
+}
+
+// Update opens a database read/write transaction and executes the function f
+// with the transaction passed as a parameter.  After f exits, if f did not
+// error, the transaction is committed.  Otherwise, if f did error, the
+// transaction is rolled back.  If the rollback fails, the original error
+// returned by f is still returned.  If the commit fails, the commit error is
+// returned.
+func (db *db) Update(f func(tx walletdb.ReadWriteTx) error) error {
+	apply := func(stm STM) error {
+		return f(newReadWriteTx(stm))
+	}
+
+	return RunSTM(db.cli, apply)
+}
+
+func (db *db) BeginReadWriteTx() (walletdb.ReadWriteTx, error) {
```

Comment:
Missing a `godoc` comment here and below.

Reply:
done